onnxruntime/core/common/threadpool.cc

```diff
@@ -377,6 +377,13 @@ ThreadPool::ThreadPool(Env* env,
   assert(degree_of_parallelism >= 1);
   if (degree_of_parallelism >= 2) {
     int threads_to_create = degree_of_parallelism - 1;
+
+    if (!thread_options_.affinity.empty()) {
```

Comment:
It is reasonable, but necessary?
What's the difference between keeping core 0 out of scope and the core that is in the end?
Is there any supportive performance numbers?